selfdrive/ui/qt/widgets/keyboard.cc

```diff
@@ -7,35 +7,37 @@
 
 #include "keyboard.hpp"
 
+const int DEFAULT_WIDTH = 1;
+const int SPACEBAR_WIDTH = 3;
```

Comment:
stretch isn't really width. why did we offset the default by 1? shouldn't 0 work the same?

Reply:
Oh I thought it was "weight". I'll check 0

Reply:
Hm 0 doesn't really work(https://imgur.com/a/TPJrTtZ). I call it width since we are in a HBoxLayout.